### DISCUSSION TITLE
Looking for Open source Turborepo Run Summary

### DISCUSSION THREAD

**Discussion by guillaumewuip** [vercel/community#3089] [Feature Question]
### Question

When using Turborepo without Vercel, we cannot benefit from the [Turborepo Run Summary](https://vercel.com/changelog/turborepo-run-summary-is-now-available) feature. Instead we are left with the `--summarize` flag and manual JSON inspection, which can be very cumbersome.

As anyone heard of a similar tool, but open source?

**Comment by ItsWendell** [comment]
I ran into this discussion as I was also looking for a nice way to display Turbo summaries in GitHub's Job Summaries.

I've made an experimental / draft of an action here in this gist, it also shows instructions on how to add this to your repository.

https://gist.github.com/ItsWendell/dbc2cd74dc8ae2d7bfa3a50afa32f079
